ios/MullvadREST/RESTNetworkOperation.swift

```diff
@@ -281,8 +281,23 @@ extension REST {
             // Increment retry count.
             retryCount += 1
 
-            // Retry immediatly if retry delay is set to never.
-            guard retryStrategy.retryDelay != .never else {
+            switch retryStrategy.retryDelay {
```

Comment:
`WaitStrategy` could produce iterator for each specific strategy. Then operation wouldn't have to account for each specific stragedy.

Reply:
Done.